### PR TITLE
add some native contracts

### PR DESCRIPTION
**Pull request by zhiqiangxu**

### CODE REVIEW

**Comment by christianxiao** [comment]
In file ```cross_chain_manager/entrance.go```, at the end of function ```ImportOuterTransfer```, this code is removed from poly old version:
```
//NOTE, you need to store the tx in this
err = MakeTransaction(native, txParam, chainID)
```
this code stores tx in cacheDb and notify an makeProof event, is that ok to remove this?

**Comment by zhiqiangxu** [comment]
> In file `cross_chain_manager/entrance.go`, at the end of function `ImportOuterTransfer`, this code is removed from poly old version:
> 
> ```
> //NOTE, you need to store the tx in this
> err = MakeTransaction(native, txParam, chainID)
> ```
> 
> this code stores tx in cacheDb and notify an makeProof event, is that ok to remove this?

thanks, it was missed.

File: contracts/native/cross_chain_manager/common/param.go [hunk 1]
```diff
@@ -0,0 +1,244 @@
+package common
```
Comment (tanZiWen) [inline contracts/native/cross_chain_manager/common/param.go:1]: add license

File: contracts/native/cross_chain_manager/common/utils.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package common
```
Comment (tanZiWen) [inline contracts/native/cross_chain_manager/common/utils.go:1]: add license

File: contracts/native/cross_chain_manager/utils.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package cross_chain_manager
```
Comment (tanZiWen) [inline contracts/native/cross_chain_manager/utils.go:1]: add license

File: contracts/native/governance/node_manager/abi_test.go [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package node_manager
```
Comment (tanZiWen) [inline contracts/native/governance/node_manager/abi_test.go:1]: add license

File: contracts/native/governance/node_manager/states.go [hunk 5]
```diff
@@ -0,0 +1,179 @@
+package node_manager
```
Comment (tanZiWen) [inline contracts/native/governance/node_manager/states.go:1]: add license

File: contracts/native/governance/node_manager/utils.go [hunk 6]
```diff
@@ -0,0 +1,152 @@
+package node_manager
```
Comment (tanZiWen) [inline contracts/native/governance/node_manager/utils.go:1]: add license

File: contracts/native/governance/relayer_manager/relayer_manager_test.go [hunk 7]
```diff
@@ -0,0 +1,172 @@
+package relayer_manager
```
Comment (tanZiWen) [inline contracts/native/governance/relayer_manager/relayer_manager_test.go:1]: add license

File: contracts/native/governance/relayer_manager/utils.go [hunk 8]
```diff
@@ -0,0 +1,172 @@
+/*
```
Comment (tanZiWen) [inline contracts/native/governance/relayer_manager/utils.go:1]: add license

File: contracts/native/cross_chain_manager/btc/states.go [hunk 9]
```diff
@@ -0,0 +1,427 @@
...
+			}
+		}
+	}
+	if pass == 1 {
+		return selection, sum, fee
+	}
+	return nil, 0, 0
+}
+
+//func (selector *CoinSelector) mixUpUtxos() {
```
Comment (tanZiWen) [inline contracts/native/cross_chain_manager/btc/states.go:276]: remove unused code

**Pull request merged**